Mathlib/NumberTheory/Sylvester.lean

```diff
@@ -0,0 +1,98 @@
+/-
+Copyright (c) 2025 Walter Moreira, Joe Stubbs. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Walter Moreira, Joe Stubbs
+-/
+import Init.Data.List.Nat.Pairwise
```

Comment:
You don't need to import files from `Init` manually
```suggestion
```

Reply:
That's good to know, thanks!